tinygrad/ops/ops_cpu.py

```diff
@@ -2,20 +2,18 @@
 from ..tensor import Function
 
 class CPUBuffer(np.ndarray):
-  log = lambda x: np.log(x)
-  exp = lambda x: np.exp(x)
-  relu = lambda x: np.maximum(x, 0)
-  expand = lambda x,shp: np.broadcast_to(x, shp)
-  permute = lambda x,order: x.transpose(order)
-  type = lambda x,tt: x.astype(tt)
-  custompad = lambda x,padding: np.pad(x, padding)
-  def amax(x, *args, **kwargs):
-    return np.amax(x, *args, **kwargs)
-  def toCPU(x):
-    return x
+  def toCPU(x): return x
+  def log(x): return np.log(x)
+  def exp(x): return np.exp(x)
+  def relu(x): return np.maximum(x, 0)
+  def type(x, tt): return x.astype(tt)
+  def custompad(x, padding): np.pad(x, padding)
```

Comment:
Does this one need a return?

Reply:
I wasn't sure we could spare the extra characters. The dangers of code golfing caught up with me!